### PR TITLE
New recipe for public-wrappers.

### PR DESCRIPTION
**Pull request by tesujimath**

### CODE REVIEW

File: recipes/public-wrappers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - pytoml
+
+test:
+  imports:
+    - public_wrappers
+  commands:
+    - configure-public-wrappers --help
+
+about:
+  home: https://github.com/tesujimath/{{ name }}
```
Comment (synapticarbors) [inline recipes/public-wrappers/meta.yaml]: It's a style thing, but could you not use jinja templating in urls here and below?

File: recipes/public-wrappers/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  entry_points:
+    - configure-public-wrappers = public_wrappers.__main__:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools_scm
+    - setuptools_scm_git_archive
+    - future
```
Comment (synapticarbors) [inline recipes/public-wrappers/meta.yaml]: I don't think that `future` is a `build` requirement (as `setup.py` doesn't look like it imports it). If that's the case, please remove it from the `build` section.

File: recipes/public-wrappers/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/tesujimath/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (synapticarbors) [inline recipes/public-wrappers/meta.yaml]: Can you switch to the new standard of using `pip`. You'll need to add `pip` as a `build` requirment:

https://conda-forge.org/docs/meta.html#use-pip

File: recipes/public-wrappers/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+{% set version = "0.2.1" %}
+{% set sha256 = "f2173dc480ea8b2baf59b2811d19783ea86005aabf2a7fdbb284a2a752d04a6a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/tesujimath/{{ name }}/archive/v{{ version }}.tar.gz
```
Comment (synapticarbors) [inline recipes/public-wrappers/meta.yaml]: From the error or CircleCI, it doesn't look like you can build this from the github archive tarball. We prefer that you don't pull source from a git repo:

https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

I'm not sure what the best solution is here. Is it feasible to submit the project to pypi?

File: recipes/public-wrappers/meta.yaml (hunk 1 above)
Comment (tesujimath) [reply recipes/public-wrappers/meta.yaml]: OK, fine.

File: recipes/public-wrappers/meta.yaml (hunk 2 above)
Comment (tesujimath) [reply recipes/public-wrappers/meta.yaml]: Ah yes, removed.

File: recipes/public-wrappers/meta.yaml (hunk 3 above)
Comment (tesujimath) [reply recipes/public-wrappers/meta.yaml]: OK, missed that before sorry.

File: recipes/public-wrappers/meta.yaml (hunk 4 above)
Comment (tesujimath) [reply recipes/public-wrappers/meta.yaml]: Yep, project is on PyPI.  I'll try that as the origin for the tarball.

**Pull request merged**